Feature/#255 change owner of notebooks to jupyter in entrypoint py (#265)
exasol/ai-lab: doc/changes/changes_1.1.0.md

Reviewer: We will release version 2.0.0, please adapt the version accordingly

# AI-Lab 1.1.0 released tbd

> Code name: Second Release

Reviewer: ```suggestion
Code name: Use non-privileged user for running JupyterLab
```

## Summary

The following changes are especially important if you are using the AI-Lab's Docker Edition and are [mounting a volume](../user_guide/docker/managing-user-data.md) containing your private notebook files and the  [Secure Configuration Storage](../user_guide/docker/secure-configuration-storage.md)  (SCS) into the AI-Lab's Docker container.

Major changes

1. The mount-point for Jupyter notebook files and the SCS has moved from `/root/notebooks` to `/home/jupyter/notebooks`.

> 2. Some of the notebooks have been updated, especially the Cloud storage notebook.

Reviewer: ```suggestion
2. Some of the notebooks have been updated, especially the Cloud Storage Extension notebook.
```

tbd

This release changes the mount-point for Jupyter notebook files and the [Secure Configuration Storage](../user_guide/docker/secure-configuration-storage.md) from `/root/notebooks` to `/home/jupyter/notebooks`.

> So in case you are mounting a volume containing your private notebook files into the AI-Lab's Docker container, then please change your scripts to use the new mounting point as described in the [User Guide](../user_guide/docker/docker-usage.md#creating-a-docker-container-for-the-ai--lab-from-the-ai-lab-docker-image).

Reviewer: ```suggestion
So in case you are mounting a volume containing your private notebook files into the AI-Lab's Docker container, then please change your commands to use the new mount point as described in the [User Guide](../user_guide/docker/docker-usage.md#creating-a-docker-container-for-the-ai--lab-from-the-ai-lab-docker-image).
```

Please note that the libraries and dependencies of the AI-Lab have changed since release 1.0.0 which can break some of the cells in old notebooks. So in case of problems, please do **not mount** notebook files created by an older version of the AI-Lab but create a new empty volume.

> 

Reviewer: Cloud Storage NB hat sich geändert

Author: Add to changes file that some notebooks have changed and in case users mount volume they will find the latest notebooks in folder `/home/jupyter/notebook-defaults`

Author: See latest push
